### PR TITLE
Bus work

### PR DESCRIPTION
**Pull request by Vanie2020**
2019 world value update & ACS metric update

### CODE REVIEW

File: config/data.js [hunk 1]
```diff
@@ -1762,6 +1762,7 @@ module.exports = {
       y_2016: 2.5,
       y_2017: 2.3,
       y_2018: 2.3,
+      y_2019: 2.4,
```
Comment (timstallmann) [inline config/data.js:1765]: @Vanie2020 where is the BUSWK world value?

Comment (Vanie2020) [reply config/data.js:1765]: Ill push it again. should be in this branch

**Review by Vanie2020 — COMMENTED** [review]
@timstallmann here you go!

File: config/data.js (hunk 1 above)
Comment (timstallmann) [reply config/data.js:1766]: great, thx!

**Comment by timstallmann** [comment]
@BiSeye looks good to me!

**Pull request merged**